cloudflare/resource_cloudflare_device_posture_integration_test.go

```diff
@@ -22,6 +22,16 @@ func TestAccCloudflareDevicePostureIntegrationCreate(t *testing.T) {
 		os.Setenv("CLOUDFLARE_API_TOKEN", "")
 	}
 
+	token := os.Getenv("WORKSPACE_ONE_API_TOKEN")
```

Comment:
let's move this to a PreCheck like https://github.com/cloudflare/terraform-provider-cloudflare/blob/3dd18dc80e5954329fc55867a217a4f0937a9309/cloudflare/provider_test.go#L117-L124 and call it in the test itself after `testAccessAccPreCheck`.